requirements.txt

```diff
@@ -19,6 +19,10 @@ mypy
 trustme
 cryptography
 coverage
+httpx
+pytest-asyncio
+async_generator; python_version < '3.7'
```

Comment:
Should we pin some of these, perhaps HTTPX and `pytest-asyncio` to the latest minor versions?

Reply:
yep I pinned both